ground/gcs/src/plugins/config/configautotunewidget.h

```diff
@@ -27,28 +27,36 @@
 #ifndef CONFIGAUTOTUNE_H
 #define CONFIGAUTOTUNE_H
 
-#include "ui_autotune.h"
-#include "ui_autotuneproperties.h"
-#include "ui_autotunesliders.h"
-#include "ui_autotunefinalpage.h"
 #include "../uavobjectwidgetutils/configtaskwidget.h"
 #include "extensionsystem/pluginmanager.h"
 #include "uavobjectmanager.h"
 #include "uavobject.h"
 #include "actuatorsettings.h"
 #include "stabilizationsettings.h"
 #include "systemident.h"
-#include <QWidget>
+
+#include <QChart>
+#include <QLineSeries>
 #include <QTimer>
+#include <QWidget>
 #include <QWizardPage>
 #include <QtNetwork/QNetworkReply>
 
+QT_CHARTS_USE_NAMESPACE
+
+#include "ui_autotune.h"
+#include "ui_autotunebeginning.h"
+#include "ui_autotuneproperties.h"
+#include "ui_autotunesliders.h"
+#include "ui_autotunefinalpage.h"
 #include "configgadgetwidget.h"
 
 struct AutotunedValues
 {
     bool valid;
 
+    QByteArray *data;
```

Comment:
No, the default initialiser will be used, which is an empty container (sizeof = sizeof(void *)). Assignment operator https://github.com/qt/qtbase/blob/d1ff9b419da5e3ace68479b02582222486ec9703/src/corelib/tools/qbytearray.h#L177 which simply assigns the d pointer from the other container (i.e. it costs the same as assigning a pointer). This is the way Qt containers are designed to be used.